src/sota_tools/CMakeLists.txt

```diff
@@ -15,33 +15,36 @@ set(SOTA_TOOLS_LIB_SRC
     server_credentials.cc
     treehub_server.cc)
 
-if (BUILD_SOTA_TOOLS)
-    set(GARAGE_TOOLS_VERSION "${AKTUALIZR_VERSION}")
-    set_property(SOURCE garage_tools_version.cc PROPERTY COMPILE_DEFINITIONS GARAGE_TOOLS_VERSION="${GARAGE_TOOLS_VERSION}")
-    add_library(sota_tools_static_lib STATIC ${SOTA_TOOLS_LIB_SRC})
-    target_include_directories(sota_tools_static_lib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${PROJECT_SOURCE_DIR} ${GLIB2_INCLUDE_DIRS})
-endif (BUILD_SOTA_TOOLS)
-
+# do not link tests with libaktualizr
+list(REMOVE_ITEM TEST_LIBS aktualizr_lib)
```

Comment:
I thought that the garage tools used some of functionality from libaktualizr, like Utils and such. Is that linked in still somehow, or how does that work now?

Reply:
It's now included in `sota_tools_lib` which links statically with libaktualizr. I've pushed some new changes to make it clearer.